_includes/primary-nav-toggle.html

```diff
@@ -0,0 +1,15 @@
+<button type="button" class="nav-btn c-primary-nav__item">
+  <span class="visuallyhidden">
+    Primary Navigation Toggle
+  </span>
+  <span class="toggle-open">
+    <svg class="c-icon c-primary-nav__icon" aria-hidden="true">
```

Comment:
thanks for the feedback! I can look into updating #1. For #2 the icon is ornamental so I purposely left out the alt info so the AT can skip it.